Sources/SwifterAuth.swift

```diff
@@ -114,6 +115,63 @@ public extension Swifter {
             }
         }, failure: failure)
     }
+  
+    func authorizeSSO(success: SSOTokenSuccessHandler?,
+                      failure: FailureHandler? = nil) {
+        guard let client = client as? SwifterAppProtocol else {
+            let error = SwifterError(message: "SSO not supported AppOnly client",
+                                     kind: .invalidClient)
+            failure?(error)
+            return
+        }
+        
+        let urlScheme = "swifter-\(client.consumerKey)"
+        
+        let nc = NotificationCenter.default
+        self.swifterCallbackToken = nc.addObserver(forName: .swifterSSOCallback, object: nil, queue: .main) { notification in
+            self.swifterCallbackToken = nil
+            let url = notification.userInfo![CallbackNotification.optionsURLKey] as! URL
```

Comment:
Are we always certain that userInfo is non-nil? Should probably use an optional-check here.

Reply:
Fixed!
https://github.com/mattdonnelly/Swifter/pull/271/commits/3392e47e03941c8841617d126c9cbccebb7fff66